### PR TITLE
String Comparison

### PR DESCRIPTION
**Pull request by aparnamittal**
## Related Issue
(#1105)

## Proposed Changes
Added a file comparing two strings .
Most string comparison algorithms compare the strings character by character. C++ provides us some built in operators like == and != to compare strings along with strcmp and compare functions.

image

## Additional Info
- Any additional information or context

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

# Algorithm:
1. Declare two strings.
2. take the input of two strings.
3. comparing the two strings we can use any of the three ways mentioned below:

i) strcmp() inbuilt function
ii) compare() inbuilt function, 
iii) C++ relational operators ( ‘==’, ‘!=’)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
1.   when strings are equal
image

2.  when strings are not equal
image

### CODE REVIEW

**Comment by Mrjoy832** [comment]
You need to follow particular guidelines for your README , please follow
these guidelines [click here](https://github.com/Lakhankumawat/LearnCPP#to-add-programs-using-stl-follow-same-approach-as--algorithms) 
also you can see how your README should look like [see here](https://github.com/Lakhankumawat/LearnCPP/blob/main/S-SortingAlgorithms/README.md) 
for more please go through these [here](https://youtu.be/7UgC3HDETRA)

**Comment by Mrjoy832** [comment]
> @Mrjoy832  sir I have made the readme according to the guidelines mentioned but other people who have contributed to the string file have not added their readme.

Let me check

**Comment by aparnamittal** [comment]
@Mrjoy832 @Lakhankumawat  sir pls review my PR

**Comment by Lakhankumawat** [comment]
Typo in file name 'StringComparison'

**Comment by Lakhankumawat** [comment]
Typo in file name 'StringComparison'

File: S-Strings/StringComparision.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#include <iostream>
```
Comment (Lakhankumawat) [inline S-Strings/StringComparision.cpp:1]: Change the cpp file name too

**Pull request closed without merging**